gel/_internal/_codegen/_models/_pydantic.py

```diff
@@ -5661,6 +5669,90 @@ def get_ptr_type(
 
         return result
 
+    def _write_std_and(self):
+        self.write(textwrap.dedent("""\
+            def and_(  # type: ignore [override, unused-ignore]
+                a: base_shapes.bool | builtins.bool | type[base_shapes.bool],
+                b: base_shapes.bool | builtins.bool | type[base_shapes.bool],
+                *args: Sequence[
+                    base_shapes.bool | builtins.bool | type[base_shapes.bool
+                ]],
+            ) -> type[base_shapes.bool]:
+                match a:
```

Comment:
is using match here necessary? You're hardcoding this implementation, why not just if?